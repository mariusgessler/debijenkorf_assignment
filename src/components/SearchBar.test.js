import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import SearchBar from './SearchBar';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<SearchBar />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders correctly", () => {
  const component = renderer.create(<SearchBar/>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
