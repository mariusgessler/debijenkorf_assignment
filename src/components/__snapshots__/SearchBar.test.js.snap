// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="search-bar"
>
  <div
    className="search-bar__container"
  >
    <form
      className="search-bar__form"
    >
      <div
        className="search-bar__input-container"
      >
        <input
          aria-label="Search input"
          autoComplete="off"
          className="search-bar__input"
          id="searchInput"
          name="searchInput"
          placeholder="Zoeken"
          type="search"
        />
        <button
          className="search-bar__button search-bar__button--search"
          id="searchButton"
          type="submit"
          value="Search"
        >
          <svg
            className="search-bar__icon search-bar__icon--search"
            title="Search"
          >
            search.svg
          </svg>
        </button>
      </div>
    </form>
  </div>
</div>
`;
